notebooks/data-sources/TestGrid/testgrid_metadata_EDA.ipynb

```diff
@@ -283,7 +283,7 @@
      "name": "stdout",
```

Comment:
<p>You could change "response" and "response1" to "openshift" and "jetpack"; it gets a little confusing later on. However, given my earlier comment, I'm not sure if using a non Openshift grid is the best thing to do at the moment. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>